CHANGELOG.md

```diff
@@ -1,10 +1,224 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-Last releases: [[1.15.0] - 2022-06-30](#1150---2022-06-30), [[1.14.0] - 2022-01-19](#1140---2022-01-19), [[1.13.0] - 2021-10-28](#1130---2021-10-28) and [[1.12.0] - 2020-11-26](#1120---2020-11-26).
+Last releases: [[1.16.0] - 2023-01-31](#1160---2023-01-31), [[1.15.0] - 2022-06-30](#1150---2022-06-30), [[1.14.0] - 2022-01-19](#1140---2022-01-19), [[1.13.0] - 2021-10-28](#1130---2021-10-28) and [[1.12.0] - 2020-11-26](#1120---2020-11-26).
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [1.16.0] - 2023-01-31
+
+This release is compatible with Coq versions 8.13, 8.14, 8.15, 8.16 and 8.17.
+
+The contributors to this version are:
+
+Cyril Cohen, Enrico Tassi, Erik Martin-Dorel, Georges Gonthier,
+Yoshihiro Ishiguro, Julien Puydt, Kazuhiko Sakaguchi, Laurent Théry,
+Mireia G. Bedmar, Pierre-Marie Pédrot, Pierre Roux, Pierre Pomeret-Coquot, Quentin Vermande,
+Reynald Affeldt, Takafumi Saikawa, Tragicus, Yoshihiro Imai
```

Comment:
```suggestion
Mireia G. Bedmar, Pierre-Marie Pédrot, Pierre Roux, Pierre Pomeret-Coquot, Quentin Vermande,
Reynald Affeldt, Takafumi Saikawa, Yoshihiro Imai
```
Quentin Vermande and @Tragicus are the same person. We should alias them in the mailmap.